modules/apps/commerce/commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/portlet/action/EditAccountEntryCommerceShippingOptionMVCActionCommand.java

```diff
@@ -109,6 +109,10 @@ protected void doProcessAction(
 			return null;
 		}
 
+		if (commerceShippingFixedOption == null) {
```

Comment:
can't we merge this with the previous IF statement?
if (commerceShippingFixedOption == null) {
 if(commerceShippingOptionAccountEntryRel != null){
   ......deleteCommerceShippingOptionAccountEntryRel....
 }
return null;
}